app/Resources/views/base.html.twig

```diff
@@ -154,6 +154,15 @@
             <script>
                 $(document).ready(function() {
                     hljs.initHighlightingOnLoad();
+
+                    $('body').on('submit', 'form[data-confirmation-message]', function (event) {
```

Comment:
I generally use ``$(document)`` for delegation, which avoids a DOM query for ``'body'`` (even though jquery can probably optimize the DOM query for the body).

It also avoids the need to wait for DOM ready for that.